Modules/_json.c

```diff
@@ -1866,25 +1812,54 @@ PyDoc_STRVAR(module_doc,
 static int
 _json_exec(PyObject *module)
 {
```

Comment:
I dislike calling get_json_state(module) multiple times. I would prefer to have a `_jsonmodulestate *state = get_json_state(module);` variable and reuse it.